### ISSUE TITLE
Can't move handles when VoiceOver is turned on on iOS

### ISSUE THREAD

**Issue by chrisy-lili** [airbnb/rheostat#117]
When VioceOver is turned on on my iPhone, neither dragging the handles directly or swiping up/down (as the instruction announced by VoiceOver) can move the handles. This is reproducible in the examples too. Not sure how to fix this accessibility issue. Any suggestions would be appreciated.

**Comment by backwardok** [comment]
I'm a little bit at a loss because even the [example](http://w3c.github.io/aria-practices/examples/slider/slider-1.html) in the [authoring guide](http://w3c.github.io/aria-practices/#slider) doesn't work on VO on an iPhone or on TalkBack on my Android phone. I'd argue that it's a bug in operating systems in that [they haven't implemented this behavior](https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/AccessibilityRoles.html), but the reality is that even if they were to implement this feature soon, it would still be broken in previous versions. My general suggestion has been to also implement text fields for the different values, but that doesn't really help this library.
